spec/lib/one_time_password/validator_spec.rb

```diff
@@ -0,0 +1,117 @@
+require "rails_helper"
+
+RSpec.describe OneTimePassword::Validator do
+  subject { described_class.new(one_time_passcode, generated_at) }
+
+  let!(:one_time_passcode) { OneTimePassword::Generator.new.code }
+  let!(:generated_at) { Time.now }
```

Comment:
not blocking but don't see why these can't be lazy ie `let` instead of `let!`

Reply:
because I have a spec that uses `travel` to move 20 minutes into the future, to test for passcode expiry.